kubemarine/testsuite.py

```diff
@@ -138,6 +154,18 @@ def get_summary(self, show_description=False, show_hint=False, show_minimal=Fals
 
         return output
 
+    def check_color(self):
+        if self.args.get('log') is not None:
+            for argsgroup in self.args.get('log'):
+                if argsgroup[0].split(";")[0] != 'stdout':
+                    continue
+                for argument in argsgroup[0].split(";"):
+                    if argument == 'colorize=false':
+                        return False
+            return True
+        else:
+            return True
```

Comment:
Alternatively, more simpler:
```suggestion
    def check_color(self):
        for handler in cluster.log.handlers:
            if isinstance(handler, log.StdoutHandler) and handler.formatter.colorize:
                return True
        return False
```
provided that either cluster, or cluster.log is passed to TestCase instance.